Makefile

```diff
@@ -21,82 +21,114 @@ endif
 
 all: manager
 
-# Create a local docker registry, start kind cluster, and install Orkestra
+## Create kind cluster with local registry and prepopulate the local registry.
+setup-dev-env: kind-create prepopulate-kind-registry
```

Comment:
Why `setup-dev-env` and not just `setup`?

Reply:
Since we are doing this only for developers just calling it setup won't be sufficient.